test/ui/commands/test_update.py

```diff
@@ -0,0 +1,205 @@
+import os
+
+from mediafile import MediaFile
+
+from beets import library, util
+from beets.test import _common
+from beets.test.helper import BeetsTestCase, IOMixin
+from beets.ui.commands.update import update_items
+from beets.util import MoveOperation, syspath
+
+
+class UpdateTest(IOMixin, BeetsTestCase):
+    def setUp(self):
+        super().setUp()
+
+        # Copy a file into the library.
+        item_path = os.path.join(_common.RSRC, b"full.mp3")
+        item_path_two = os.path.join(_common.RSRC, b"full.flac")
+        self.i = library.Item.from_path(item_path)
+        self.i2 = library.Item.from_path(item_path_two)
+        self.lib.add(self.i)
+        self.lib.add(self.i2)
+        self.i.move(operation=MoveOperation.COPY)
+        self.i2.move(operation=MoveOperation.COPY)
+        self.album = self.lib.add_album([self.i, self.i2])
+
+        # Album art.
+        artfile = os.path.join(self.temp_dir, b"testart.jpg")
+        _common.touch(artfile)
+        self.album.set_art(artfile)
+        self.album.store()
+        util.remove(artfile)
```

Comment:
Just import `remove` from `utils` specifically (I'm aware it was like this originally but best to update regardless).